.github/workflows/build_and_push_images.yml

```diff
@@ -0,0 +1,53 @@
+name: "Build and Push Images"
+
+on:
+  push:
+    paths:
+      - "app/**"
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    strategy:
+      matrix:
+        tag:
```

Comment:
nit: to avoid a confusion with git tag I would rename it to something else e.g. `app`?

Reply:
They are called "tag" by docker as well, but I can go with app for sure.

Reply:
fixed